docs/en-us/user/references/xml/dubbo-config-center.md

```diff
@@ -0,0 +1,19 @@
+| highest-priority  | config.highestPriority | boolean             | optional     | true             | Configuration items from the configuration center have the highest priority, meaning the local configuration items are overwritten. | 2.7.0+ |
+| namespace        | config.namespace       | string              | optional     | dubbo            | Often used for multi-tenant isolation, the actual meaning varies depending on the configuration center. <br />For example:<br />zookeeper - Environment isolation, default `dubbo`；<br />apollo - Distinguish between sets of configurations for different domains and use them by default `dubbo` and `application` | 2.7.0+ |
+| cluster          | config.cluster         | string              | optional     |                  | The meaning varies according to the configuration center selected.<br />For example, in apollo it's used to distinguish between different configuration clusters | 2.7.0+ |
+| group            | config.group           | string              | optional     | dubbo            | The meaning varies according to the configuration center selected.<br />nacos - Isolate different configuration sets<br />zookeeper - Isolate different configuration sets | 2.7.0+ |
+| check            | config.check           | boolean             | optional     | true             | Whether to terminate application startup when the configuration hub connection fails.                     | 2.7.0+ |
+| config-file       | config.configFile      | string              | optional     | dubbo.properties | The key mapped to the global level profile<br />zookeeper - DefaultPath/dubbo/config/dubbo/dubbo.properties<br />apollo - The dubbo.properties key in dubbo namespace | 2.7.0+ |
+| timeout          | config.timeout         | integer             |          | 3000ms           | Gets the configured timeout | 2.7.0+ |
+| username         |                        | string              |          |                  | the username if the configuration center requires validation<br />Apollo is not yet enabled           | 2.7.0+ |
+| password         |                        | string              |          |                  | password if configuration center needs to do check<br />Apollo is not yet enabled             | 2.7.0+ |
+| parameters       |                        | Map<string, string> |          |                  | Extended parameters to support customized configuration parameters for different configuration centers | 2.7.0+ |
+| include-spring-env |                        | boolean             | optional     | false            | With the Spring framework support, `true` automatically reads the configuration from the Spring Environment<br />Read by default<br />Configuration with key `dubbo.properties`<br />PropertySource with key `dubbo.properties` | 2.7.0+ |
```

Comment:
```suggestion
| include-spring-env |                        | boolean             | optional     | false            | With the Spring framework support, when the value is `true`, it will automatically reads the configuration from the Spring Environment<br />Read by default<br />Configuration with key `dubbo.properties`<br />PropertySource with key `dubbo.properties` | 2.7.0+ |
```